.github/workflows/main.yml

```diff
@@ -30,7 +30,7 @@ jobs:
       - uses: subosito/flutter-action@v1
         with:
           channel: ${{ matrix.flutter_version }}
-      - run: ./tool/flutter_ci_script.sh
+      - run: source tool/flutter_ci_script.sh
         continue-on-error: ${{ matrix.allow_failure }}
```

Comment:
I believe continue-on-error defaults to false, so this line can be taken out.

https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idcontinue-on-error